### ISSUE TITLE
Add Support X86 on big platforms

### ISSUE THREAD

**Issue by toor1245** [toor1245/cpu_features.NET#10]
In this task it necessary to create libs to x86 for Linux and Windows OS and add integration to github actions

**Comment by Mizux** [comment]
you may want to add aarch64 macOS aka Apple M1...

ps: https://github.com/Mizux/dotnet-native/blob/2c4737b1eb29bc3a93911a632520774fe1b74e2f/cmake/dotnet.cmake#L28-L30
:wink:

**Comment by toor1245** [comment]
@Mizux, yes it would be great to add `osx-arm64`, but the current project is a full `cpu_features` binding, so first of all I'm looking forward to the day we add arm64 support to macOS :smile:. I didn't find emulation about apple m1, only bitrise CI/CD [bitrise-apple-silicon-m1-stacks](https://devcenter.bitrise.io/en/infrastructure/build-stacks/apple-silicon-m1-stacks.html)
in hobby plan of pricing, I see free use of M1 https://www.bitrise.io/pricing, but for me this is not a convenient docs/UI, maybe we need to check if we can integrate into cpu_features, but again there will be a mess with CI/CD services

also thanks a lot for your project https://github.com/Mizux/dotnet-native this project helps me a lot to understand how to integrate native code in .NET
